deployments/liqo/templates/clusterconfig.yaml

```diff
@@ -30,8 +30,10 @@ spec:
     enableDiscovery: true
     name: MyLiqo
     port: 6443
-    service: _liqo._tcp
+    service: _liqo_api._tcp
     ttl: 90
+  authConfig:
+    allowEmptyToken: true
```

Comment:
Why are we setting this flag to true?

Reply:
We set it to true in order to have the same "automatic peer" behavior. With an empty token allowed the clusters will join automatically as before the AuthService

We can disable it to not allow automatic join, but each time the user has to do manual actions to provide the token, this choice makes simpler the first user experience

Reply:
Gotcha, thanks!